.circleci/config.yml

```diff
@@ -77,16 +78,34 @@ anchor_verify_and_test_python: &anchor_verify_and_test_python
         <<: *anchor_check_generation
     # Generated Logging client is broken: https://github.com/googleapis/gapic-generator/issues/2209.
     - run:
-        name: Prepare for Pubsub testing.
+        name: Run Showcase Generated Unit Tests.
         command: |
-          echo 'export TEST_API="pubsub-v1"' >> $BASH_ENV
+          nox --session "unit(py='2.7')" \
+              --noxfile /tmp/workspace/gapic-generator/showcase/python/nox.py
+          nox --session "unit(py='3.5')" \
+              --noxfile /tmp/workspace/gapic-generator/showcase/python/nox.py
+          nox --session "unit(py='3.6')" \
+              --noxfile /tmp/workspace/gapic-generator/showcase/python/nox.py
+        when: always
+    - run:
+        name: Run Showcase Integration Tests.
+        command: |
+          nox --session "showcase(py='2.7')" \
+              --noxfile /tmp/workspace/gapic-generator/showcase/python/nox.py
+          nox --session "showcase(py='3.5')" \
+              --noxfile /tmp/workspace/gapic-generator/showcase/python/nox.py
+          nox --session "showcase(py='3.6')" \
```

Comment:
You can restrict showcase to 2.7 and 3.7.

Reply:
Done!

Reply:
I think we can only use 3.6 because the latest image of nox is not compatible some stuff we are using in circle and that image only has python 3.6.